### ISSUE TITLE
Granting database role SNOWFLAKE.CORTEX_USER to another role fails

### ISSUE THREAD

**Issue by siku92** [Titan-Systems/titan#170]
Hi, 

I am trying to grant the database role SNOWFLAKE.CORTEX_USER to another role with the following config:

```
role_grants:
  - role: SNOWFLAKE.CORTEX_USER
    to_role: CORTEX_ANALYST_ROLE
```

But I get the following error:
`failed to execute sql, [GRANT ROLE SNOWFLAKE.CORTEX_USER TO ROLE CORTEX_ANALYST_ROLE]`

I would expect the following sql to be executed:
`grant database role SNOWFLAKE.CORTEX_USER to role CORTEX_ANALYST_ROLE`

titan-core version: 0.10.21
Role used: ACCOUNTADMIN

Did I miss anything in my config?

Thansks!

**Comment by teej** [comment]
This is related to https://github.com/Titan-Systems/titan/issues/164 which should be fixed in the future v0.11 release

**Comment by teej** [comment]
Fixed in [v0.11.0](https://github.com/Titan-Systems/titan/releases/tag/v0.11.0). Use the new `DatabaseRoleGrant` resource.
